curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/chaining-if-else-statements.portuguese.md

```diff
@@ -23,21 +23,21 @@ tests:
     testString: 'assert(code.match(/if/g).length > 3, "You should have at least four <code>if</code> statements");'
   - text: Você deve ter pelo menos uma declaração de <code>return</code>
     testString: 'assert(code.match(/return/g).length >= 1, "You should have at least one <code>return</code> statement");'
-  - text: <code>testSize(0)</code> deve retornar &quot;minúsculo&quot;
+  - text: <code>testSize(0)</code> deve retornar &quot;Muito pequeno&quot;
```

Comment:
I like the changes, but the description says one thing and the tests says another. You should also change the test assertion and error strings so that the tests match the description.

Reply:
@thefrontendwizard Are you suggesting a change to the actual test or just the error message argument of the assert?

Reply:
These error messages are misleading. We have to address that. They say that the function should return the a certain translated value, but they check the value in english.